index.html

```diff
@@ -0,0 +1,85 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Great Idea!</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="index.css">
+
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
+
+  <body>
+
+      <html>
```

Comment:
No need for this html tag here....only goes at the beginning and end of an html page.

```
< html >
    < head >
        < title > Here is a website title < /title >
        < link  link to your stylesheet >
    < /head >
    < body >
       All the stuff that physically goes on the website goes here
    < /body >
< /html >

```

p.s. Don't pay any mind to the spacing in between the carats and the letters - I wanted you to be able to see how it would look on the page...if I left no spaces it would have treated it as code. 😛